packages/ui/src/components/va-input/components/VaTextarea/VaTextarea.vue

```diff
@@ -82,10 +88,11 @@ export default defineComponent({
 
     return {
       textarea,
-      computedHeight: computed(() => ({
+      computedStyle: computed(() => ({
         minHeight: rowHeight.value * props.minRows + 'px',
-        maxHeight: rowHeight.value * props.maxRows + 'px',
+        maxHeight: props.maxRows && (rowHeight.value * props.maxRows + 'px'),
         height: height.value + 'px',
+        resize: props.autosize && 'none',
```

Comment:
Would be great if you can add prop description to [our docs](https://vuestic.dev/en/ui-elements/input#api) as well.

Reply:
Probably we have some problems with our doc app, because in translations we have this description:
<img width="847" alt="Screenshot 2021-12-28 at 14 23 03" src="https://user-images.githubusercontent.com/55198465/147561414-de991e7e-3868-4780-992a-f323705ca72c.png">

Reply:
@RVitaly1978, try to add `name: 'VaInput'` to component options. That should make a trick. 

Vue class component auto generates component name from its class. Define component not able to make that, so we need to provide component name manually. Would be great to have component name as ESLint rule.

Reply:
fixed it in `va-input` and its inner components